src/mixins/stackable.js

```diff
@@ -13,28 +13,33 @@ export default {
   computed: {
     activeZIndex () {
       const content = this.stackElement || this.$refs.content
-      if (!this.isActive) {
-        // Return current zindex if not active
+      // Return current zindex if not active
+      if (!this.isActive) return getZIndex(content)
 
-        return getZIndex(content)
-      }
-      // Return max current z-index (excluding self) + 2 (2 to leave room for an overlay below, if needed)
-
-      return this.getMaxZIndex((this.stackExclude || (() => [content]))()) + 2
+      // Return max current z-index (excluding self) + 2
+      // (2 to leave room for an overlay below, if needed)
+      return this.getMaxZIndex(this.stackExclude || [content]) + 2
     }
   },
   methods: {
     getMaxZIndex (exclude = []) {
       const base = this.stackBase || this.$el
-      // start with lowest allowed z-index or z-index of base component's element, whichever is greater
+      // start with lowest allowed z-index or z-index of
+      // base component's element, whichever is greater
       const zis = [this.stackMinZIndex, getZIndex(base)]
       // get z-index for all active dialogs
-      const activeElements = document.getElementsByClassName(this.stackClass)
-      for (const activeElement of activeElements) {
+      const activeElements = [...document.getElementsByClassName(this.stackClass)]
```

Comment:
This line is what actually fixes the issue, you can still use `for..of` after doing this.